internal/oci/container.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/containers/libpod/pkg/cgroups"
 	"github.com/containers/storage/pkg/idtools"
 	specs "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/pkg/errors"
```

Comment:
We can use the golang `errors` package instead of using this.

Reply:
I prefer pkg/errors, it's more comprehensive

Reply:
I think we will have to check across the repo if we are using anything beyond https://blog.golang.org/go1.13-errors. But this is okay for now since we already are using it.